grafana/make_grafana_webpack.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
+#
+# Copyright (c) 2019 Red Hat.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+
+[ $# -ne 1 ] && echo "Usage: $0 version" && exit 1
+
+# grafana version (must be tagged on github.com/grafana/grafana as "v$VER")
+VER=$1
+BUILDDIR=`mktemp -d buildXXXXXX`
+
+which npm
+[ $? -ne 0 ] && echo Please install \"npm\" package && exit 1
+which wget
+[ $? -ne 0 ] && echo Please install \"wget\" package && exit 1 
+
+# get src tree and set cwd
+echo Fetching pristine upstream git tagged branch for grafana version v$VER ...
+git clone https://github.com/grafana/grafana grafana-$VER
+cd grafana-$VER
```

Comment:
Check for git as well, or is it safe to assume it's installed? (See my comment above about BuildRequires too)

Reply:
fixed